src/App.tsx

```diff
@@ -2,13 +2,15 @@ import Header from './Header/Header';
 import Aside from './Aside/Aside';
 import WeekCalendar from './WeekCalendar/WeekCalendar';
 import styles from './index.module.css';
+import React from 'react';
+import { useState } from 'react';
 
 function App() {
-  const currentDate = new Date();
+  const [currentDate, setCurrentDate] = useState(new Date());
```

Comment:
```suggestion
  const [selectedDate, setSelectedDate] = useState(new Date());
```
or
```suggestion
  const [weekCalendarDate, setWeekCalendarDate] = useState(new Date());
```
or
```suggestion
  const [calendarDate, setCalendarDate] = useState(new Date());
```

because it is current date only until none of the arrows have been clicked